docs/guides/migration-guides.md

```diff
@@ -199,6 +199,19 @@ uppy.use(Dropbox, {
 
 ### `@uppy/aws-s3-multipart`
 
+#### Companion’s options (`companionUrl`, `companionHeaders`, and `companionCookieRules`) are renamed to more generic names (`endpoint`, `headers`, and `cookieRules`)
```

Comment:
This is placed inside the `2.x` to `3.x` docs. You can move it correctly when #5206 is merged.